lib/msal-browser-poc/src/error/ClientBrowserAuthError.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I wonder if they errors need to be split by browser or non browser.

for instance a common/errors/ConfigurationError, could be used by both and then browser specific classes might life in msal-browser such as IframeError (or something)

Reply:
That's the idea, I will flesh this out more in future PRs.